<!DOCTYPE html>
<html>
    <head> 
        <meta charset="utf-8">  
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <title>ALSER Lab @ Georgia State University</title>
        <meta name="description" content="ALSER Lab @ Georgia State University, Led by Prof. Mohammed Alser">
        <script>var clicky_site_ids = clicky_site_ids || []; clicky_site_ids.push(101230698);</script>
        <script async src="//static.getclicky.com/js"></script>
        <!-- Open Graph -->
        <!-- Bootstrap & MDB -->
        <link rel="stylesheet" href="/assets/css/bootstrap.css">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous" />

        <!-- Fonts & Icons -->
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"  integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous">
        <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">
        <!-- Styles -->
        <link rel="icon" type="image/png" href="/assets/img/GSU-29x29.png">
        <link rel="apple-touch-icon" type="image/png" href="/assets/img/GSU-29x29.png">
        <link rel="stylesheet" href="/assets/css/main.css">
        <link rel="canonical" href="/index.html">
        <!-- jQuery -->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
        <style>.carousel-inner > .carousel-item > img { width:100%; height:400px; } </style>
    </head>
    <body class="fixed-top-nav" style="padding-bottom: 0px;">
        <!-- Header -->
        <header>
            <!-- Nav Bar -->
            <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
                <div class="container">
                    <a class="navbar-brand title font-weight-lighter" href="../index.html">
                        <!-- <span class="font-weight-bold">ALSER Lab</span>-->
                        <img style="position:relative; left:0px" src="/assets/img/logo/AlserLabHeaderLogo.png" height="40" width="auto" class="rounded float-right" alt="...">
                    </a>
                    <!-- Navbar Toggle -->
                    <!-- 
<button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
<span class="sr-only">Toggle navigation</span>
<span class="icon-bar top-bar"></span>
<span class="icon-bar middle-bar"></span>
<span class="icon-bar bottom-bar"></span>
</button>
-->
                    <div class="collapse navbar-collapse text-right" id="navbarNav">
                        <ul class="navbar-nav ml-auto flex-nowrap">
                            <!-- About -->
                            <li class="nav-item">
                                <a class="nav-link" href="/index.html">
                                    About
                                    <span class="sr-only">(current)</span>
                                </a>
                            </li>
                            <!-- Blog -->
                            <!--<li class="nav-item ">
<a class="nav-link" href="blog/index.html">
Blog
</a>
</li>
<li class="nav-item ">
<a class="nav-link" href="projects/index.html">
Projects
</a>
</li>-->

                            <li class="nav-item ">
                                <a class="nav-link" href="/people/index.html">
                                    People
                                </a>
                            </li>
                            <li class="nav-item active">
                                <a class="nav-link" href="/research/index.html">
                                    Research
                                </a>
                            </li>
                            <li class="nav-item ">
                                <a class="nav-link" href="/teaching/index.html">
                                    Teaching
                                </a>
                            </li>
                            <li class="nav-item ">
                                <a class="nav-link" href="/publications/index.html">
                                    Publications
                                </a>
                            </li>
                            <li class="nav-item ">
                                <a class="nav-link" href="/news/index.html">
                                    News
                                </a>
                            </li>
                            <li class="nav-item ">
                                <a class="nav-link" href="/opportunities/index.html">
                                    Work with us
                                </a>
                            </li>
                            <li class="nav-item ">
                                <a class="nav-link" href="/contact/index.html">
                                    Contact
                                </a>
                            </li>
                            <!--
<div class="toggle-container">
<a id="light-toggle">
<i class="fas fa-moon"></i>
<i class="fas fa-sun"></i>
</a>
</div>
-->
                        </ul>
                    </div>
                </div>
            </nav>
        </header>
        <!-- Content -->
        <div class="container mt-5">
            <!--<div id="homeid" class="col-sm-8">-->
            <div class="post">
                <header class="post-header">
                    <p><br /></p>
                    <center>
                        <h3 class="post-title">
                            <span>Our Research</span>
                        </h3>
                    </center>
                    <p><br /></p>
                </header>
                <article>
                    <div class="row justify-content-sm-center">
                        <div class="col-sm-12 mt-3 mt-md-0">
                            <img class="img-fluid rounded z-depth-1" src="/assets/img/research/Research1.gif" alt="" title="Genome analysis" />
                        </div>
                    </div>
                    <div class="caption">
                       Genome analysis in real life.
                    </div>
                    
                     <p>The purpose of computing is always to gain insights, not numbers (as Richard Hamming said). Today, there is a dire need to gain insights and observations much more efficiently than ever before, especially after witnessing both unprecedented growth in data volume and the start of the Angstrom era of semiconductors. Bioinformatics is one of the scientific disciplines that necessitate such efficient computation due to the complexity of data and problems that bioinformatics should solve timely. In bioinformatics, we develop methods and build architectures to solve biological problems. Understanding and analyzing biological data, such as genomes, is the foundation of many scientific and medical discoveries in biomedicine and other life sciences through its rapidly growing presence in clinical medicine, outbreak tracing, and understanding of pathogens and urban microbial communities.
                    </p>
                     <p>Our understanding of genomic molecules today is affected by the ability of modern computing technology to quickly and accurately determine an individual’s entire genome. To computationally analyze an organism’s genome, the DNA molecule must first be converted to digital data in the form of a string over an alphabet of four letters or base-pairs (bp), commonly denoted by A, C, G, and T. The four letters in the DNA alphabet correspond to four chemical bases, adenine, cytosine, guanine, and thymine, respectively, which make up a DNA molecule. After more than 7 decades of continuous attempts (since 1945, the first attempt to read the full content of <a href="https://www.nobelprize.org/uploads/2018/06/sanger-lecture.pdf">Insulin protein and later was granted a Nobel prize</a>), there is still no sequencing technology that can read a DNA molecule in its entirety. As a workaround, sequencing machines generate randomly sampled subsequences of the original genome sequence, called reads. The resulting reads lack information about their order and corresponding locations in the complete genome. Software tools, collectively known as genome analysis tools, are used to reassemble read fragments back into an entire genome sequence and infer genomic variations that make an individual genetically different from another.
                    </p>

                    <div class="row justify-content-sm-center">
                        <div class="col-sm-12 mt-3 mt-md-0">
                            <img class="img-fluid rounded z-depth-1" src="/assets/img/research/Research2.gif" alt="" title="Solving the puzzle" />
                        </div>
                    </div>
                    <div class="caption">
                       Building back our complete DNA sequence after sequencing can be very, very difficult!
                    </div>
                    <p>Since sequencing genomic data provides randomized fragments of the actual genome sequence, we need to reassemble back these fragments. Similar to solving jigsaw puzzles, we match each read sequence to one or more possible locations within a reference genome (i.e., a representative genome sequence for a particular species) based on the similarity between the read and the reference sequence segment at that location. When the sample donor is known, we can obtain a reference genome for the donor species from public reference databases (e.g., RefSeq, PATRIC, and Ensembl). It is more challenging when we aim to identify differences (somatic variations) between two sets of reads, for example,  try to understand what genetic variations cause a cancer by comparing reads sequenced from normal and cancerous cells of the same person. It can be much more challenging when we do NOT know the sample donor. This requires comparing the reads to ALL known reference genomes (4 TB of data in RefSeq alone). For example, when collecting a genomic sample from a train seat, we expect to have a collection of bacterial and viral species (and probably others) present in the sample. For that, the detection of species in the sample highly depends on what species we have already known before and already stored in reference databases.
                    </p>
                    <p>When we match each read sequence to each potential location within a reference genome, the bases in a read may, unfortunately, not be identical to the bases in the reference genome at the location that the read actually comes from. These differences may be due to (1) sequencing errors (with a rate in the range of 0.1–20% of the read length, depending on the sequencing technology), and (2) genetic differences that are specific to the individual organism’s DNA and may not exist in the reference genome. We must tolerate such differences during similarity checks, which makes it even more challenging. When we build back the sequenced genome, we aim to identify the possible genetic differences between the reference genome and the sequenced genome. Genetic differences include small variations that are less than 50 bp, such as single nucleotide polymorphisms (SNPs) and small insertions and deletions (indels). Genetic differences can also be larger than 50 bp variations, known as structural variations, which are caused by chromosome-scale changes in a genome. For example, the insertion of about 600,000-base long region has been observed in some chromosomes.
                    </p>
                    <div class="row justify-content-sm-center">
                        <div class="col-sm-12 mt-3 mt-md-0">
                            <img class="img-fluid rounded z-depth-1" src="/assets/img/research/Research3.png" alt="" title="achievements" />
                        </div>
                    </div>
                    <div class="caption">
                       An overview of some of our achievements (along with collaborators) in various genomic analyses
                    </div>
                    <p>We developed new algorithms and architected different solutions and designs that are now an integral part of genomic pipelines and used to solve challenging biological problems. As most bioinformatics applications are implemented as computer programs, there is always an urgent and constant need to improve bioinformatics software tools that can benefit significantly from algorithmic optimizations and adapting new computing techniques. We believe that rethinking existing bioinformatics pipelines to redefine roles played by each computational step can provide transformational benefits. Our long-term goal is to develop and enable truly intelligent genomic analyses, where genomic data are analyzed efficiently at a truly population scale. The lag between data generation and obtaining analysis results may pose problems in rapid clinical diagnosis, for example, in identifying infectious agents, choosing the correct antibiotics for sepsis and hospital infections, identifying genetic disorders in critically ill infants, and tracking outbreaks that spread quickly in the population as evidenced by recent outbreaks. The problem is exacerbated in developing countries and during crises where access to data centers is even more limited. We are thrilled to continue improving bioinformatics applications, especially those that are essential components of biomedical and clinical applications. We are eager to collaborate with other scientists to achieve practical and realizable complete solutions using existing and emerging technologies, such as near-data processing, quantum computing, neuromorphic computing, specialized architectures, and reconfigurable fabrics. Looking forward, we are excited about working with medical domain experts to identify and optimize other challenging clinical applications that can benefit from my expertise in performance improvement.
                    </p>
                </article>
        </div>
    </div>
<!-- Footer -->

<footer class="text-center text-lg-start text-dark" style="background-color: #ECEFF1" >
    <!-- Grid container -->
    <div class="container p-4">
        <!--Grid row-->
        <div class="row mt-4">
            <!--Grid column-->
            <div class="col-lg-4 col-md-12 mb-4 mb-md-0">
                <h5 class="text-uppercase mb-4">Connect with us</h5>

                <p class="mt-4" style="font-size: 26px;">
                    <a href="https://twitter.com/mealser"><i class="fab fa-twitter"></i></a>
                    <a href="https://scholar.google.com.tr/citations?user=wnYVkJIAAAAJ&hl=en"><i class="fas fa-graduation-cap"></i></a>
                    <a href="https://github.com/mealser"><i class="fab fa-github"></i></a>
                    <a href="https://www.linkedin.com/in/mohammed-alser-06bbabb1/"><i class="fab fa-linkedin"></i></a>
                    <!--<a href="mailto:alser@usc.edu"><i class="fas fa-envelope"></i></a>-->
                </p>

                <ul class="fa-ul" style="margin-left: 1.65em;text-align: left;">
                    <li class="mb-3">
                        <span class="fa-li"><a href=""><i class="fas fa-home"></i></a></span>
                        <span class="ms-2">ALSER Lab, 
                            <br>
                            Department of Computer Science, 
                            <br>
                            Georgia State University, 
                            <br>
                            25 Park Place, Suite 700, 
                            <br>
                            Atlanta, GA 30303</span>
                    </li>
                    <li class="mb-3">
                        <span class="fa-li"><a href=""><i class="fas fa-envelope"></i></a></span>
                        <span class="ms-2">alser.lab.bio@gmail.com</span>
                    </li>
                    <li class="mb-3">
                        <span class="fa-li"><a href=""><i class="fas fa-phone"></i></a></span>
                        <span class="ms-2">+1(404)-413-5700</span>
                    </li>
                </ul>
            </div>
            <!--Grid column-->

            <!--Grid column-->
            <div class="col-lg-4 col-md-6 mb-4 mb-md-0">
                <h5 class="text-uppercase mb-4 pb-1">Visit us</h5>
                <ul class="fa-ul" style="margin-left: 1.65em;">
                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3317.210118674171!2d-84.38775170000001!3d33.755235799999994!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x88f50386501ea7e7%3A0x7273991ca97e4234!2s25%20Park%20Pl%20NE%2C%20Atlanta%2C%20GA%2030303!5e0!3m2!1sen!2sus!4v1722203237213!5m2!1sen!2sus" width="250" height="250" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
                </ul>
            </div>
            <!--Grid column-->

            <!--Grid column-->
            <div class="col-lg-4 col-md-6 mb-4 mb-md-0">
                <h5 class="text-uppercase mb-4">Our online visitors</h5>
                <a href='https://clustrmaps.com/site/1c0nd'  title='Visit tracker'><img src='//clustrmaps.com/map_v2.png?cl=ffffff&w=200&t=tt&d=8izY-Hk3PAHhRnkbARB_EJ8dcZq4WwqThHCKdzJSolk&co=0095ff&ct=000000'/>
                </a>
            </div>
            <!--Grid column-->
        </div>
        <!--Grid row-->
    </div>
    <!-- Grid container -->
    <!-- Copyright -->
    <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2)" >
        © 2025 Copyright: ALSER Lab, <a class="text-dark" href="https://AlserLab.github.io/">AlserLab.github.io</a>, All rights reserved.
    </div>
    <!-- Copyright -->
</footer>

<!-- Footer -->


</body>
<!-- Bootsrap & MDB scripts -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.4/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script>
<!-- Mansory & imagesLoaded -->
<script defer src="https://unpkg.com/masonry-layout@4/dist/masonry.pkgd.min.js"></script>
<script defer src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
<script defer src="assets/js/mansory.js" type="text/javascript"></script>
<!-- Load Common JS -->
<script src="assets/js/common.js"></script>
</html>
